OpenRA.Mods.TS/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -72,40 +72,43 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0x47, "palet02" },
 			{ 0x48, "palet03" },
 			{ 0x49, "palet04" },
-			/*
-			{ 0x4A, "lobrdg01" },
-			{ 0x4B, "lobrdg02" },
-			{ 0x4C, "lobrdg03" },
-			{ 0x4D, "lobrdg04" },
-			{ 0x4E, "lobrdg05" },
-			{ 0x4F, "lobrdg06" },
-			{ 0x50, "lobrdg07" },
-			{ 0x51, "lobrdg08" },
-			{ 0x52, "lobrdg09" },
-			{ 0x53, "lobrdg10" },
-			{ 0x54, "lobrdg11" },
-			{ 0x55, "lobrdg12" },
-			{ 0x56, "lobrdg13" },
-			{ 0x57, "lobrdg14" },
-			{ 0x58, "lobrdg15" },
-			{ 0x59, "lobrdg16" },
-			{ 0x5A, "lobrdg17" },
-			{ 0x5B, "lobrdg18" },
-			{ 0x5C, "lobrdg19" },
-			{ 0x5D, "lobrdg20" },
-			{ 0x5E, "lobrdg21" },
-			{ 0x5F, "lobrdg22" },
-			{ 0x60, "lobrdg23" },
-			{ 0x61, "lobrdg24" },
-			{ 0x62, "lobrdg25" },
-			{ 0x63, "lobrdg26" },
-			{ 0x64, "lobrdg27" },
-			{ 0x65, "lobrdg28" },
+
+			{ 0x4A, "lobrdg_b" }, // lobrdg01
+			{ 0x4B, "lobrdg_b" }, // lobrdg02
+			{ 0x4C, "lobrdg_b" }, // lobrdg03
+			{ 0x4D, "lobrdg_b" }, // lobrdg04
+			{ 0x4E, "lobrdg_b_nw" }, // lobrdg05
+			{ 0x4F, "lobrdg_b_se" }, // lobrdg06
+			{ 0x50, "lobrdg_b" }, // lobrdg07
+			{ 0x51, "lobrdg_b_nw" }, // lobrdg08
+			{ 0x52, "lobrdg_b_se" }, // lobrdg09
+			{ 0x53, "lobrdg_a" }, // lobrdg10
+			{ 0x54, "lobrdg_a" }, // lobrdg11
+			{ 0x55, "lobrdg_a" }, // lobrdg12
+			{ 0x56, "lobrdg_a" }, // lobrdg13
+			{ 0x57, "lobrdg_a_sw" }, // lobrdg14
+			{ 0x58, "lobrdg_a_ne" }, // lobrdg15
+			{ 0x59, "lobrdg_a" }, // lobrdg16
+			{ 0x5A, "lobrdg_a_sw" }, // lobrdg17
+			{ 0x5B, "lobrdg_a_ne" }, // lobrdg18
+			{ 0x5C, "lobrdg_r_se" }, // lobrdg19
+			{ 0x5D, "lobrdg_r_se" }, // lobrdg20
+			{ 0x5E, "lobrdg_r_nw" }, // lobrdg21
+			{ 0x5F, "lobrdg_r_nw" }, // lobrdg22
+			{ 0x60, "lobrdg_r_ne" }, // lobrdg23
+			{ 0x61, "lobrdg_r_ne" }, // lobrdg24
+			{ 0x62, "lobrdg_r_sw" }, // lobrdg25
+			{ 0x63, "lobrdg_r_sw" }, // lobrdg26
+			{ 0x64, "lobrdg_b" }, // lobrdg27
```

Comment:
My spec from #8916 was to have these two dead versions as their own actor types.  This (1) lets us handle the special case adjacency rendering that only the dead state needs, and (2) will let us eventually remove the hacky zombie / resurrection logic that is now only used for bridges.